ipaserver/install/cainstance.py

```diff
@@ -388,6 +388,21 @@ def configure_instance(self, host_name, dm_password, admin_password,
             self.ca_type = x509.ExternalCAType.GENERIC.value
         self.external_ca_profile = external_ca_profile
         self.random_serial_numbers = random_serial_numbers
+        dn = DN("cn=config,cn=ldbm database,cn=plugins,cn=config")
+        try:
+            entry = api.Backend.ldap2.get_entry(dn)
+        except errors.NotFound:
+            ldap_backend = 'bdb'
+        else:
+            ldap_backend = entry.get('nsslapd-backend-implement', ['bdb'])[0]
+
+        if ldap_backend != 'bdb' and not random_serial_numbers:
+            # override selection for lmdb due to VLV performance issues.
+            logger.info(
+                'Random serial numbers are required for the %s backend',
```

Comment:
May be clarify the message to say that not only RSNs are required but also were forced on.